src/main/java/com/aws/greengrass/telemetry/SystemMetricsEmitter.java

```diff
@@ -24,8 +36,10 @@
     private static final int MB_CONVERTER = 1024 * 1024;
     private static final int PERCENTAGE_CONVERTER = 100;
     public static final String NAMESPACE = "SystemMetrics";
+#if !ANDROID
     private static final SystemInfo systemInfo = new SystemInfo();
     private static final CentralProcessor cpu = systemInfo.getHardware().getProcessor();
+#endif
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

Syntax error on token &#34;Invalid Character&#34;, @ expected

This file contains syntax errors and can&#39;t be analyzed by all of CodeGuru Reviewer&#39;s detectors. Fix the syntax errors and run the repository analysis or submit the pull request again.